### ISSUE TITLE
isRemoteNotificationsEnabled always returns false on iOS 10

### ISSUE THREAD

**Issue by Dakuan** [dpa99c/cordova-diagnostic-plugin#121]
``` es6
window.cordova.plugins.diagnostic.isRemoteNotificationsEnabled((enabled) => {
  // enabled is always false
})
```

was working a charm on iOS 8 & 9, but always false on iOS 10, regardless of what the user has accepted.

**Comment by vivekpd15** [comment]
+1

**Comment by dpa99c** [comment]
iOS 10 has made some fundamental changes to user notifications. I need to update this plugin to handle them. So this is a bug in the sense that the plugin doesn't yet support the new methods introduced in iOS 10.

**Comment by dpa99c** [comment]
Fix published in v3.2.0 to npm

**Comment by vivekpd15** [comment]
I tried making a new build using this update, but I am being greeted with this error:

.../cordova_project/platforms/ios/vImmune/Plugins/cordova.plugins.diagnostic/Diagnostic.m:13:9: fatal error: 'UserNotifications/UserNotifications.h' file not found
#import <UserNotifications/UserNotifications.h>
        ^
1 error generated.

I am using Cordova 6.2.0. Any ideas why this is happening?

**Comment by dpa99c** [comment]
@vivekpd15 This error is being raised because the new UserNotifications framework (added in XCode 8 / iOS 10) is not being found.
The latest version of this plugin (v3.2.0) [injects this framework dependency](https://github.com/dpa99c/cordova-diagnostic-plugin/blob/master/plugin.xml#L81) into the XCode project, so uninstalling the old plugin version from your project and installing the latest version should be enough to satisfy this dependency.
However, it seems for some reason in your case, this has not happened. So open your Cordova iOS project in XCode and check if the UserNotifications framework is included. If not, add it manually.
And of course you must be using the latest XCode 8, because UserNotifications framework was only added for iOS 10 in this version.
